coremltools/converters/mil/frontend/torch/test/test_torch_conversion_api.py

```diff
@@ -264,6 +264,32 @@ def test_func_dummy(context, inputs):
         # Cleanup the test
         del _TORCH_OPS_REGISTRY.name_to_func_mapping["test_func_dummy"]
 
+    @staticmethod
+    def test_is_torch_fx_node_supported():
+        class Model(torch.nn.Module):
+            def __init__(self):
```

Comment:
nit: If your constructor is just calling `super().__init__()`, then you don't need it; If the sub class doesn't have a constructor, then the base constructor is called automatically.

Reply:
Good point 👍 Removed empty constructor as suggested